### PR TITLE
Science magboots

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

These science magboots currently replace the normal magboots obtained in the magboot research on the technology tree. They are functionally the same as normal ones, and have the same name. 

I made the sprite use the same colour palette as the new GORILLA gauntlets as to fit the theme of science gear.

image

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image
image

**Changelog**

:cl: Ubaser
- add: Magboots produced by science now have new sprites.

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml [hunk 1]
```diff
@@ -56,6 +56,22 @@
   - type: StealTarget
     stealGroup: ClothingShoesBootsMagAdv
 
+- type: entity
+  parent: ClothingShoesBootsMag
+  id: ClothingShoesBootsMagSci
+  name: magboots
+  description: Magnetic boots, often used during extravehicular activity to ensure the user remains safely attached to the vehicle.
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml:63]:
```suggestion
```
inherited from the base magboots

File: Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml [hunk 2]
```diff
@@ -56,6 +56,22 @@
...
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/magboots-science.rsi
+    state: icon
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml:67]: you probably should just remove the state definition here and just inherit the layers from the base one. Having both a state and layers defined is kinda weird and i'm not sure how it works.

**Review by EmoGarbage404 — DISMISSED** [review]

**Comment by lzk228** [comment]
cuties!

**Comment by K-Dynamic** [comment]
Now let's make these magboots slightly faster than engineering ones :)

**Comment by Emisse** [comment]
no

**Comment by Kadeo64** [comment]
drippy

**Comment by PuroSlavKing** [comment]
they refused to add magboots for security service, but approved for scientists...

**Pull request merged**